{"id":983,"details":"\n  \n  \u003ctable\u003e\n    \u003ctr\u003e\n      \u003ctd\u003eRegion\u003c/td\u003e\n\u003ctd\u003eAuvergne-RhÌ«ne-Alpes\u003c/td\u003e\n    \u003c/tr\u003e\n    \u003ctr\u003e\n\u003ctd\u003eDepartment\u003c/td\u003e\n\u003ctd\u003ePuy-de-DÌ«me\u003c/td\u003e\n\u003c/tr\u003e\n    \n    \n  \u003c/table\u003e\n  ","patient":"30 Years Old Male Patient From Auvergne Rhì«Ne Alpes, Puy De Dì«Me","source":"unknown_source","status":"unreported","confirmed_at":"2020-03-08T00:00:00.000Z","created_at":"2020-03-16T10:13:05.870Z","updated_at":"2020-03-16T10:42:32.055Z","infected_by":"unknown_infection","number":709,"urls":["https://github.com/lperez31/coronavirus-france-dataset"],"recovered_at":null,"gender":"male","age":30,"nationality":"unidentified_nationality","symptomatic_at":null,"slug":"france-case-709-30-years-old-male-patient-from-auvergne-rhi-ne-alpes-puy-de-di-me","suggested_url":"france-case-709-30-years-old-male-patient-from-auvergne-rhi-ne-alpes-puy-de-di-me","hospital_stay_length_in_days":null,"days_from_symptoms_to_confirmation":null,"without_symptoms":false,"reporting_country":"fr"}